client/results.js

```diff
@@ -0,0 +1,43 @@
+const url = 'http://localhost:5000/search?search=pie';
```

Comment:
only question around why url is declared at the start but then redeclared in the functions?
otherwise all looks good to me